examples/rllib/cooperative_merge.py

```diff
@@ -130,7 +130,7 @@
 )
 
 if __name__ == "__main__":
-    ray.init(num_cpus=PARALLEL_ROLLOUTS, redirect_output=False)
+    ray.init(num_cpus=PARALLEL_ROLLOUTS+1, redirect_output=False)
```

Comment:
should we start using `N_CPUS` instead of `PARALLEL_ROLLOUTS`? So that people don't ask for 8 for example when their laptop don't have 9 cpus

Reply:
Yeah, that's clearer. I'll change them all to say that.